tests/regression_tests/microxs/test.py

```diff
@@ -40,14 +40,17 @@ def model():
 
     settings = openmc.Settings()
     settings.particles = 1000
-    settings.inactive = 10
-    settings.batches = 50
+    settings.inactive = 5
+    settings.batches = 10
```

Comment:
Why change the number of batches?

Reply:
As you probably know, running the entire test suite in CI takes much longer than we would like. Since the results from this test needed to be updated, I figured I would try to minimize the time as while by reducing the number of batches.